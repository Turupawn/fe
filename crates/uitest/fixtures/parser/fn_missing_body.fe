fn foo(x: u8, y: u64) -> u8

fn bar<T>() asdf

fn baz(x: u8) -> u8 {
    return 10
}
