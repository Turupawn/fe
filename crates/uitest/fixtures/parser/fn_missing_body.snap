---
source: crates/uitest/tests/parser.rs
expression: diags
input_file: crates/uitest/fixtures/parser/fn_missing_body.fe
---
error[1-0001]: expected `{` or `where`
  ┌─ fn_missing_body.fe:1:28
  │
1 │ fn foo(x: u8, y: u64) -> u8
  │                            ^ expected `{` or `where`

error[1-0001]: unexpected syntax while parsing function definition
  ┌─ fn_missing_body.fe:3:13
  │
3 │ fn bar<T>() asdf
  │             ^^^^ unexpected


